### ISSUE TITLE
[1.12.2] Dedicated server crash on world load: NoClassDefFoundError IICProxy

### ISSUE THREAD

**Issue by Brycey92** [BluSunrize/ImmersiveEngineering#3638]
### Description of the issue:
My dedicated server crashes when it was loads the world. This happens every time I start the server, as far as I can tell. See the crashlog for details.

### Crashlog:
[crash-2020-01-18_13.56.16-server.txt](https://gist.github.com/Brycey92/8ec6d328d0e618fecad62438e20b906e)

### Versions & Modlist:
Forge 14.23.5.2838
Immersive Engineering 0.12 - 92
Immersive Cables 1.3.2
Immersive Petroleum 1.1.9
Industrial Wires 1.7-36

I do have SpongeForge installed, but as far as I could tell, it's not part of the cause of the crash. Let me know if I should test without it installed anyway.

**Comment by malte0811** [comment]
Some mod is including classes from IE in their own file, which causes the crash. We can't really do anything about that. But something else is severely broken: [this line](https://gist.github.com/Brycey92/8ec6d328d0e618fecad62438e20b906e#file-gistfile1-txt-L75) in the crash report shows that it failed to load the classes for generating the modlist for the crash report as well.

**Comment by Brycey92** [comment]
Thanks for the quick response! I'll look into both issues.

**Comment by Brycey92** [comment]
I unzipped every single mod I have, and none of them are including any files from IE.

I eventually deleted the iicproxies NBT array from the IE world save data, and received this even weirder crash report: https://gist.github.com/Brycey92/8ec6d328d0e618fecad62438e20b906e

What I *did* eventually discover was that downgrading the Sponge [Player Analytics](https://ore.spongepowered.org/Rsl1122/Plan) (aka Plan) plugin by one release from 5.0-build-367 to 5.0-build-335 solved the issues completely. I think their latest update introduced some weird issue with Java's classpath or something, causing a slew of seemingly unrelated errors. Without this bugged version of Plan, the generation of the modlist for crash reports works again too.
